src/contexts/AppContext.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+
+export const AppContext = React.createContext(null);
+
+// This context provider is passed to any component requiring the context
+export const AppProvider = ({ children }) => {
+  const [isLoading, setIsLoading] = useState(false);
+  // TODO: implement lightMode
+  const [isLightMode, setIsLightMode] = useState(false);
```

Comment:
We might not need this - chakra has a nice color mode built in
https://chakra-ui.com/docs/features/color-mode